docker/Dockerfile

```diff
@@ -47,15 +47,18 @@ RUN set -x \
   && apt-get purge -y --auto-remove $buildDeps
 
 # Sane defaults for pip
-ENV PIP_NO_CACHE_DIR=off \
+ENV \
+  PIP_NO_CACHE_DIR=off \
   PIP_DISABLE_PIP_VERSION_CHECK=1 \
   # Sentry config params
   SENTRY_CONF=/etc/sentry \
   # Disable some unused uWSGI features, saving dependencies
   # Thank to https://stackoverflow.com/a/25260588/90297
   UWSGI_PROFILE_OVERRIDE=ssl=false;xml=false;routing=false \
   # UWSGI dogstatsd plugin
-  UWSGI_NEED_PLUGIN=/var/lib/uwsgi/dogstatsd
+  UWSGI_NEED_PLUGIN=/var/lib/uwsgi/dogstatsd \
+  # grpcio>1.30.0 requires this, see requirements.txt for more detail.
+  GRPC_POLL_STRATEGY=epoll1
```

Comment:
So we should remove this once Kernel version 4.5 becomes more widely available?

Reply:
I... don't think so, there's [this](https://github.com/grpc/grpc/issues/23796#issuecomment-740984462) comment about epollex not being compatible with the way celery works (grpc will say `Fork support is only compatible with the epoll1 and poll polling strategies`) and the first few sentences of [this](https://github.com/grpc/grpc/issues/17253) seem to support that:

> The newly added fork support in gRPC Python does not extend to epollex. That's somewhat understandable (although it's a bummer since epollex is the default when built from source or pip installing the source grpcio package).

However since this is untested from my end, I didn't put that in comments. It does seem most likely on 4.5+ you could use epollex but that would prevent Celery from working properly.

Reply:
Ah, sorry I didn't read the whole thread and missed the part with Celery. NVM, this looks good.

Reply:
Actually you know what, I could make these comments a bit better. Let me follow up with that, this hasn't been confirmed yet to work in prod, we've done testing in staging + s4s.